heksher/api/v1/rules.py

```diff
@@ -264,4 +264,78 @@ async def get_rule(rule_id: int, app: HeksherApp = application):
                          feature_values=rule_spec.feature_values, metadata=rule_spec.metadata)
 
 
+class InputRuleMetadata(ORJSONModel):
```

Comment:
I think the metadata apis should have their own files, just so these don't become too big